beacon-chain/monitor/service.go

```diff
@@ -40,19 +37,19 @@ type ValidatorAggregatedPerformance struct {
 // monitor service tracks, as well as the event feed notifier that the
 // monitor needs to subscribe.
 type ValidatorMonitorConfig struct {
-	StateGen          stategen.StateManager
-	TrackedValidators map[types.ValidatorIndex]interface{}
+	StateGen stategen.StateManager
 }
 
 // Service is the main structure that tracks validators and reports logs and
 // metrics of their performances throughout their lifetime.
 type Service struct {
 	config *ValidatorMonitorConfig
 
-	// monitorLock Locks access to TrackedValidators, latestPerformance, aggregatedPerformance,
+	// Locks access to TrackedValidators, latestPerformance, aggregatedPerformance,
 	// trackedSyncedCommitteeIndices and lastSyncedEpoch
-	monitorLock sync.RWMutex
+	sync.RWMutex
```

Comment:
`monitorLock` felt too verbose where the pkg name is already `monitor`, the `lock` part was self explanatory